test/utils.ts

```diff
@@ -11,3 +11,9 @@ export function repeat(func: () => void, count: number) {
 
   return results;
 }
+
+export async function executionTime(func: () => Promise<unknown>): Promise<number> {
```

Comment:
Is this used and do we need it for unit testing?

Reply:
It is used to test cache decorator.